bin/build_all

```diff
@@ -281,7 +284,10 @@ then
         docker_build docker/sawtooth-dev-javascript docker/ sawtooth-xo-tp-javascript
         docker_run sawtooth-dev-javascript
     }
-
+    build_poet_sgx() {
+        docker_build docker/sawtooth-dev-poet-sgx docker/ sawtooth-dev-poet-sgx
+        docker_run sawtooth-dev-poet-sgx
+    }
```

Comment:
Since you're not supporting installed mode, I would follow the pattern used by `build_external()` instead.